articles/overview/understanding-quantum-computing.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Understanding quantum computing
+description: What are quantum computers and how do they use the principles of quantum physics?
+author: bradben
+ms.author:  bradben
+ms.date: 04/22/2020
+ms.topic: overview
+uid: microsoft.quantum.overview.understanding
+---
+
+# Understanding quantum computing
+
+Quantum computing uses the principles of quantum physics to process information. Because of this, quantum computing requires a different approach than classical computing.  One example of this is the processor used in quantum computers.  Where classical computers use familiar silicon-based chips, quantum computers use quantum particles such atoms, ions, photons, or electrons.  
```

Comment:
I'd again suggest here that not all qubits are single particles, such as topological, quantum dot, superconducting, and other modalities of qubits.